### ISSUE TITLE
[2.3.1] Updating Youtube video via API causes data to be lost.

### ISSUE THREAD

**Issue by itwebdevgh** [magento/magento2#23194]
### Preconditions (*)
 - Magento CE 2.3.1

### Steps to reproduce (*)
1. Add a product to Magento using the API:
```
{
"product": {
"sku": "test555",
"name": "test product for video111",
"attribute_set_id": 9,
"price": 342,
"status": 1,
"visibility": 1,
"type_id": "simple",
"weight": "0.5",
"media_gallery_entries": [
{
"media_type": "external-video",
"disabled": false,
"label": "Test Video",
"types": [],
"content": {
"type": "image/png",
"name": "thumbnail.png",
"base64_encoded_data": "iVBORw0KGgoAAAANSUhEUgAAAMgAAADICAMAAACahl6sAAAAvVBMVEX///+aBgZCB1v57++dDQ305OT16Oi7WFjNg4Piubn+/Pzv2Nj89/f9+vr37Oz68vKhGBj39PirMTGnJia3Tk5ID2BPGGbDbGyeERHy7vTmwcG1R0fAZGRUHmq9XFyznL3qy8tpO33x3Nzq5O3k2+fTk5OIYpfao6OtNjZ3TImtlLhlNXlaJ3DQi4vId3eni7Kfgat6UIulISGScKDb0ODOv9XCr8qBWpLTxdjdqqrXnZ3EssyOa52qj7WymryivbirAAAH/ElEQVR4nO2ce1fiPBDGm4IILTcRREG0gq643lZ21wvr+v0/1pukhWZmUoru2wvnzO8/SfXMI0lm8mTAcRiGYRiGYRiGYRiGYRiGYRiGYRiGYRiGYZjtmZ0UHcE2NEf3/f36Wydp/Pxp4rqTm+c8Y/oKo4XQLBrW4dmTG/E0yzmyT9EcijVnTTr+euSuGfzOP75tafWFwZhMr1+GDqnkTxExboO3LwDzChyHOqSSki4U70wg5m1z/BLpkJRTySnWIUSvFg9fTogO1z0vLtxEhlSHVLK3Gp7dWXS47keRIVt5sOkQ4jBSkqDDdf8WGzahbtchE8pUDSfqcN3HokMHJOoIt66T20Qdrvuj6OANrhJ19NTGhXQM0Pb1s+jw10Adi/tlD2xbBxcg7m8nre/lVLIki6Izjpa6TYf6lWeo5N0rWILmGujo+uq1Zn+9ZR3cgJhfwpifB1BdCZS8WXTIskuWKwulo2XV4Th/oJKXVlHxr2hUgY5g9bp3pjde7wXE+xTH+3uQNFIIUMdxEI946v1AOm4OjF99PUoey50R1DHC499ArBcw1hIpCY5NHVWi432TDlLXk/HcCLpABznd/gRx3lLP4ResiC8KciV8qOMNj/9I00Fqe+szmQN1iBQddzpG8qZdwmrytgBLYroAOpZ4/BHq0BE+iAf8GKqL73JXsgd1XOHx75b4VI08xA/OYEV5d5lH9DF7h5t1nIPoJjq6e/0oUYJq40muSmo9oKOOx206VjXyKa6rTi4KU9JO0QGL26Nf6rW4Rj7DSlB9HD6fB5U50EEW8B9LXGaNvI/rKlQh56WkA3WQSQ/LwaNX9RqskfvYTsVKXvPQ0RyDoE436xiEMcEigNqprSegJA9jGOkgE/51YI0oTQmqk7NXAo1qsY91JBrVqBDAxrDjwUo5a4sbGdV9vGxRQWtG48MUCo1h9adhrZytxY2M6jFetEgHjAUVNaYxHAKr5UyN4dPNOlAxiyNB5cAhUQLrTDe72QWN6jlesCk6pBKYSPvkAahkklVVD41Rulxh0fTd8hdQaUNKZlQzv2QiwwngFMeLFQVhN6fbIJke08tfWDVnUtS3wH+TTvBLkECSrGlY3pBTPqo3M9m5gKN4uEfG/26jQxY45oq/tzxgZsZMXGHzKie89YCYCW3T5DYtb1LfSD6Mv/Pt/wrexCxN6Cp1HHOpbypfza2PFM4Ss6bP5D7LnBKBZdwUsmFuAy+MOBYoK2ZScZnFyZll3Kwwkj0EWHL5ZBxkkqNMEsm9GYFlToDtJslDgDro/wNmxGyKFJhGyHHKmYE6y37yhuVWNyUV3WSgQgELLeIhoJP6xLLgkYdEVojNQ8qACgyDHKkceDaiJ+9PekjZeXUjYL6newjo5I0KLZIMbR5SPkrSPIQBUPJJDylbdwufvLGSDR7CVzykHJUQNwR5COuz0Vc8pEzBJ+80DyHK8ch7SdExyMPXmn7OQwirDOyFbekhZawEbqIpHsKN2tqQh0S9sGKaHbGHQI4mRp2hr2r/wUPKFpQQNijRF7XIQ0rTkWejYw1uQfSY9Wjo+EcPKVvaSAkpx3XVFF7SQg+JbNhIR95NjhW4DcU9KCs+VjpgsyPxwtBlqM1DypZOmpLz1UUu0IHTDtJRRIMjSg20D0Wvaejpka0aXYQW097Y7KcocdCpknphpdARNpcZ0F4U1OxItml0CVpcayNKEEQJbXYEIB3veYVtwYMpAvXVXKfogIeXgtsaUUf8tTFkaXb0HuOWrKRmx6JAPfGxBWlrdnyPm8s8eAQruqXRId3kK0fB1uyoyuKouQw3OxavA2+xkadga3YMy/sw38PjV7GNmWtQZ7xyFWzNjquS+HbmtFKaHYsCKdlv1KEOvS3HJuLgqRytjBZgwkCEOsA1EKCYRsYErjcI0cnlOVHHXZl0yKRRTdIRppbEj8Lk38SYQiNBySqxoDa50upIUhIb1agiCcm3gXFLkDGsMY1qi5JS6iB2qsBGNTKG82xe/CRYCTaqW2kXKKUB3hDS2zlgDOfTuPhF/Ni6q9q6GpyPtRl3U779yqRVjzavXmB/YPai/eq7/H2fzzJdDsf9esPy/QIR3uXz73Jlc4ZhGIZhEgkSv8gJMaWNqiXirSe2i280F/YvfCoJ+2JLIXXBQrKn5vt9IUa+v7qtbo6WyyD2pyuj5XWgrkM7vj8UYikfTC6SiyS+YtAL3rvS/TfVh+jqY6nPj9VhzfDzguKi3QAU0lRXjN35sRBj3UGk7h8OVRvLtPRCmpWKDN6vVPRtupw8fblevIfw9O4LsQjkrLqSP7QqFfnqm3yw6HuqROLFHqw/sdMX1Zq27M1TfKkXu2MKGa4bxgNtOF5Bm2t3hByKRfRaq6rmlpTTvY5n0u4IURtUhFzujpphQvSWq87BnRHSBp6j6ubXu5ionob37bskZFxbE+a94FRlkq6+UdwZIZ5MGnS4qXJkT62UnRHizIXAzVmKTjfczXZASNRzUre52I5O/w2n/EKG60uq9jEKNX6n1DuyTNBZFq60fV1RQaummmHQlDXvUIU8Emey8vXlO6Hzy0iu+obXsXwMsBy01b5UDWNdamNeXchV29FnzfQPgRr0evpB6zoqBeE3Fy/0fuufhdeKvaX8sRN9V2A/Klx83XHbpZ97KwstvxGsD4meP2r46396LWgY1oQ3bYxKO7MYhmEYhmEYhmEYhmEYhmEYhmEYhmEYhtkF/gMowJs3YI2tmAAAAABJRU5ErkJggg=="
},
"extension_attributes": {
"video_content": {
"media_type": "external-video",
"video_provider":"youtube",
"video_url": "https://www.youtube.com/watch?v=FVFPRstvlvk",
"video_title": "Video title",
"video_description": "Video description",
"video_metadata": "Video meta"
}
}
}
]
},
"saveOptions": true
}
```

2. Send Product Video update using PUT to rest/V1/products/{SKU}:
```
{
"product": {
"sku": "test555",
"name": "test product for video111",
"attribute_set_id": 9,
"price": 342,
"status": 1,
"visibility": 1,
"type_id": "simple",
"weight": "0.5",
"media_gallery_entries": [
{
"media_type": "external-video",
"disabled": false,
"label": "Test Video",
"types": [],
"content": {
"type": "image/png",
"name": "thumbnail.png",
"base64_encoded_data": "iVBORw0KGgoAAAANSUhEUgAAAMgAAADICAMAAACahl6sAAAAvVBMVEX///+aBgZCB1v57++dDQ305OT16Oi7WFjNg4Piubn+/Pzv2Nj89/f9+vr37Oz68vKhGBj39PirMTGnJia3Tk5ID2BPGGbDbGyeERHy7vTmwcG1R0fAZGRUHmq9XFyznL3qy8tpO33x3Nzq5O3k2+fTk5OIYpfao6OtNjZ3TImtlLhlNXlaJ3DQi4vId3eni7Kfgat6UIulISGScKDb0ODOv9XCr8qBWpLTxdjdqqrXnZ3EssyOa52qj7WymryivbirAAAH/ElEQVR4nO2ce1fiPBDGm4IILTcRREG0gq643lZ21wvr+v0/1pukhWZmUoru2wvnzO8/SfXMI0lm8mTAcRiGYRiGYRiGYRiGYRiGYRiGYRiGYRiGYZjtmZ0UHcE2NEf3/f36Wydp/Pxp4rqTm+c8Y/oKo4XQLBrW4dmTG/E0yzmyT9EcijVnTTr+euSuGfzOP75tafWFwZhMr1+GDqnkTxExboO3LwDzChyHOqSSki4U70wg5m1z/BLpkJRTySnWIUSvFg9fTogO1z0vLtxEhlSHVLK3Gp7dWXS47keRIVt5sOkQ4jBSkqDDdf8WGzahbtchE8pUDSfqcN3HokMHJOoIt66T20Qdrvuj6OANrhJ19NTGhXQM0Pb1s+jw10Adi/tlD2xbBxcg7m8nre/lVLIki6Izjpa6TYf6lWeo5N0rWILmGujo+uq1Zn+9ZR3cgJhfwpifB1BdCZS8WXTIskuWKwulo2XV4Th/oJKXVlHxr2hUgY5g9bp3pjde7wXE+xTH+3uQNFIIUMdxEI946v1AOm4OjF99PUoey50R1DHC499ArBcw1hIpCY5NHVWi432TDlLXk/HcCLpABznd/gRx3lLP4ResiC8KciV8qOMNj/9I00Fqe+szmQN1iBQddzpG8qZdwmrytgBLYroAOpZ4/BHq0BE+iAf8GKqL73JXsgd1XOHx75b4VI08xA/OYEV5d5lH9DF7h5t1nIPoJjq6e/0oUYJq40muSmo9oKOOx206VjXyKa6rTi4KU9JO0QGL26Nf6rW4Rj7DSlB9HD6fB5U50EEW8B9LXGaNvI/rKlQh56WkA3WQSQ/LwaNX9RqskfvYTsVKXvPQ0RyDoE436xiEMcEigNqprSegJA9jGOkgE/51YI0oTQmqk7NXAo1qsY91JBrVqBDAxrDjwUo5a4sbGdV9vGxRQWtG48MUCo1h9adhrZytxY2M6jFetEgHjAUVNaYxHAKr5UyN4dPNOlAxiyNB5cAhUQLrTDe72QWN6jlesCk6pBKYSPvkAahkklVVD41Rulxh0fTd8hdQaUNKZlQzv2QiwwngFMeLFQVhN6fbIJke08tfWDVnUtS3wH+TTvBLkECSrGlY3pBTPqo3M9m5gKN4uEfG/26jQxY45oq/tzxgZsZMXGHzKie89YCYCW3T5DYtb1LfSD6Mv/Pt/wrexCxN6Cp1HHOpbypfza2PFM4Ss6bP5D7LnBKBZdwUsmFuAy+MOBYoK2ZScZnFyZll3Kwwkj0EWHL5ZBxkkqNMEsm9GYFlToDtJslDgDro/wNmxGyKFJhGyHHKmYE6y37yhuVWNyUV3WSgQgELLeIhoJP6xLLgkYdEVojNQ8qACgyDHKkceDaiJ+9PekjZeXUjYL6newjo5I0KLZIMbR5SPkrSPIQBUPJJDylbdwufvLGSDR7CVzykHJUQNwR5COuz0Vc8pEzBJ+80DyHK8ch7SdExyMPXmn7OQwirDOyFbekhZawEbqIpHsKN2tqQh0S9sGKaHbGHQI4mRp2hr2r/wUPKFpQQNijRF7XIQ0rTkWejYw1uQfSY9Wjo+EcPKVvaSAkpx3XVFF7SQg+JbNhIR95NjhW4DcU9KCs+VjpgsyPxwtBlqM1DypZOmpLz1UUu0IHTDtJRRIMjSg20D0Wvaejpka0aXYQW097Y7KcocdCpknphpdARNpcZ0F4U1OxItml0CVpcayNKEEQJbXYEIB3veYVtwYMpAvXVXKfogIeXgtsaUUf8tTFkaXb0HuOWrKRmx6JAPfGxBWlrdnyPm8s8eAQruqXRId3kK0fB1uyoyuKouQw3OxavA2+xkadga3YMy/sw38PjV7GNmWtQZ7xyFWzNjquS+HbmtFKaHYsCKdlv1KEOvS3HJuLgqRytjBZgwkCEOsA1EKCYRsYErjcI0cnlOVHHXZl0yKRRTdIRppbEj8Lk38SYQiNBySqxoDa50upIUhIb1agiCcm3gXFLkDGsMY1qi5JS6iB2qsBGNTKG82xe/CRYCTaqW2kXKKUB3hDS2zlgDOfTuPhF/Ni6q9q6GpyPtRl3U779yqRVjzavXmB/YPai/eq7/H2fzzJdDsf9esPy/QIR3uXz73Jlc4ZhGIZhEgkSv8gJMaWNqiXirSe2i280F/YvfCoJ+2JLIXXBQrKn5vt9IUa+v7qtbo6WyyD2pyuj5XWgrkM7vj8UYikfTC6SiyS+YtAL3rvS/TfVh+jqY6nPj9VhzfDzguKi3QAU0lRXjN35sRBj3UGk7h8OVRvLtPRCmpWKDN6vVPRtupw8fblevIfw9O4LsQjkrLqSP7QqFfnqm3yw6HuqROLFHqw/sdMX1Zq27M1TfKkXu2MKGa4bxgNtOF5Bm2t3hByKRfRaq6rmlpTTvY5n0u4IURtUhFzujpphQvSWq87BnRHSBp6j6ubXu5ionob37bskZFxbE+a94FRlkq6+UdwZIZ5MGnS4qXJkT62UnRHizIXAzVmKTjfczXZASNRzUre52I5O/w2n/EKG60uq9jEKNX6n1DuyTNBZFq60fV1RQaummmHQlDXvUIU8Emey8vXlO6Hzy0iu+obXsXwMsBy01b5UDWNdamNeXchV29FnzfQPgRr0evpB6zoqBeE3Fy/0fuufhdeKvaX8sRN9V2A/Klx83XHbpZ97KwstvxGsD4meP2r46396LWgY1oQ3bYxKO7MYhmEYhmEYhmEYhmEYhmEYhmEYhmEYhtkF/gMowJs3YI2tmAAAAABJRU5ErkJggg=="
},
"extension_attributes": {
"video_content": {
"media_type": "external-video",
"video_provider":"youtube",
"video_url": "https://www.youtube.com/watch?v=FVFPRstvlvk",
"video_title": "Video title",
"video_description": "Video description",
"video_metadata": "Video meta"
}
}
}
]
},
"saveOptions": true
}
```

### Expected result (*)

1.  Video appears on product view page/In the magento admin
![image](https://user-images.githubusercontent.com/51627977/59190271-313c1880-8b74-11e9-922a-6d186b17999e.png)

### Actual result (*)

1. Thumbnail appears, but video player does not load when play button is clicked.
![image](https://user-images.githubusercontent.com/51627977/59190331-59c41280-8b74-11e9-9398-8978fb768afd.png)

2. Fields show as empty in Magento admin:
![image](https://user-images.githubusercontent.com/51627977/59190386-81b37600-8b74-11e9-8aae-59a35c7aceb5.png)

**Comment by itwebdevgh** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @itmsenior. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @itmsenior, here is your [Magento instance](https://i-23194-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-23194-2-3-develop.instances.magento-community.engineering/admin](https://i-23194-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by itwebdevgh** [comment]
@magento it looks like for this issue to reoccur you need to be amending an already existing product. I've updated my original issue accordingly.

Reproduced on Vanilla instance.

**Comment by krishprakash** [comment]
Hi @itmsenior. 
Thank you for your report.
This is not an issue.
You need to set API key for you tube video in below path
Store > Configuration > Catalog > Product video > YouTube API Key
Also please refer issue  #22574

**Comment by itwebdevgh** [comment]
@krishprakash I have a youtube API key set. I even added this key to the Vanilla test instance.

**Comment by itwebdevgh** [comment]
@krishprakash As the other person said in #22574, could you perhaps share what you sent via swagger to the API? We have the exact same issue in 2.3.1 but only when amending products that already exist.
Could you please give an example JSON/XML that works correctly?

**Comment by krishprakash** [comment]
{
  "product": {
    "sku": "test555",
    "name": "test product for video111",
    "attribute_set_id": 9,
    "price": 342,
    "status": 1,
    "visibility": 1,
    "type_id": "simple",
    "weight": "0.5",
    "media_gallery_entries": [
        {
            "media_type": "external-video",
            "disabled": false,
            "label": "Test Video",
            "types": [],
            "content": {
                "type": "image/png",
                "name": "thumbnail.png",
                "base64_encoded_data": "iVBORw0KGgoAAAANSUhEUgAAAMgAAADICAMAAACahl6sAAAAvVBMVEX///+aBgZCB1v57++dDQ305OT16Oi7WFjNg4Piubn+/Pzv2Nj89/f9+vr37Oz68vKhGBj39PirMTGnJia3Tk5ID2BPGGbDbGyeERHy7vTmwcG1R0fAZGRUHmq9XFyznL3qy8tpO33x3Nzq5O3k2+fTk5OIYpfao6OtNjZ3TImtlLhlNXlaJ3DQi4vId3eni7Kfgat6UIulISGScKDb0ODOv9XCr8qBWpLTxdjdqqrXnZ3EssyOa52qj7WymryivbirAAAH/ElEQVR4nO2ce1fiPBDGm4IILTcRREG0gq643lZ21wvr+v0/1pukhWZmUoru2wvnzO8/SfXMI0lm8mTAcRiGYRiGYRiGYRiGYRiGYRiGYRiGYRiGYZjtmZ0UHcE2NEf3/f36Wydp/Pxp4rqTm+c8Y/oKo4XQLBrW4dmTG/E0yzmyT9EcijVnTTr+euSuGfzOP75tafWFwZhMr1+GDqnkTxExboO3LwDzChyHOqSSki4U70wg5m1z/BLpkJRTySnWIUSvFg9fTogO1z0vLtxEhlSHVLK3Gp7dWXS47keRIVt5sOkQ4jBSkqDDdf8WGzahbtchE8pUDSfqcN3HokMHJOoIt66T20Qdrvuj6OANrhJ19NTGhXQM0Pb1s+jw10Adi/tlD2xbBxcg7m8nre/lVLIki6Izjpa6TYf6lWeo5N0rWILmGujo+uq1Zn+9ZR3cgJhfwpifB1BdCZS8WXTIskuWKwulo2XV4Th/oJKXVlHxr2hUgY5g9bp3pjde7wXE+xTH+3uQNFIIUMdxEI946v1AOm4OjF99PUoey50R1DHC499ArBcw1hIpCY5NHVWi432TDlLXk/HcCLpABznd/gRx3lLP4ResiC8KciV8qOMNj/9I00Fqe+szmQN1iBQddzpG8qZdwmrytgBLYroAOpZ4/BHq0BE+iAf8GKqL73JXsgd1XOHx75b4VI08xA/OYEV5d5lH9DF7h5t1nIPoJjq6e/0oUYJq40muSmo9oKOOx206VjXyKa6rTi4KU9JO0QGL26Nf6rW4Rj7DSlB9HD6fB5U50EEW8B9LXGaNvI/rKlQh56WkA3WQSQ/LwaNX9RqskfvYTsVKXvPQ0RyDoE436xiEMcEigNqprSegJA9jGOkgE/51YI0oTQmqk7NXAo1qsY91JBrVqBDAxrDjwUo5a4sbGdV9vGxRQWtG48MUCo1h9adhrZytxY2M6jFetEgHjAUVNaYxHAKr5UyN4dPNOlAxiyNB5cAhUQLrTDe72QWN6jlesCk6pBKYSPvkAahkklVVD41Rulxh0fTd8hdQaUNKZlQzv2QiwwngFMeLFQVhN6fbIJke08tfWDVnUtS3wH+TTvBLkECSrGlY3pBTPqo3M9m5gKN4uEfG/26jQxY45oq/tzxgZsZMXGHzKie89YCYCW3T5DYtb1LfSD6Mv/Pt/wrexCxN6Cp1HHOpbypfza2PFM4Ss6bP5D7LnBKBZdwUsmFuAy+MOBYoK2ZScZnFyZll3Kwwkj0EWHL5ZBxkkqNMEsm9GYFlToDtJslDgDro/wNmxGyKFJhGyHHKmYE6y37yhuVWNyUV3WSgQgELLeIhoJP6xLLgkYdEVojNQ8qACgyDHKkceDaiJ+9PekjZeXUjYL6newjo5I0KLZIMbR5SPkrSPIQBUPJJDylbdwufvLGSDR7CVzykHJUQNwR5COuz0Vc8pEzBJ+80DyHK8ch7SdExyMPXmn7OQwirDOyFbekhZawEbqIpHsKN2tqQh0S9sGKaHbGHQI4mRp2hr2r/wUPKFpQQNijRF7XIQ0rTkWejYw1uQfSY9Wjo+EcPKVvaSAkpx3XVFF7SQg+JbNhIR95NjhW4DcU9KCs+VjpgsyPxwtBlqM1DypZOmpLz1UUu0IHTDtJRRIMjSg20D0Wvaejpka0aXYQW097Y7KcocdCpknphpdARNpcZ0F4U1OxItml0CVpcayNKEEQJbXYEIB3veYVtwYMpAvXVXKfogIeXgtsaUUf8tTFkaXb0HuOWrKRmx6JAPfGxBWlrdnyPm8s8eAQruqXRId3kK0fB1uyoyuKouQw3OxavA2+xkadga3YMy/sw38PjV7GNmWtQZ7xyFWzNjquS+HbmtFKaHYsCKdlv1KEOvS3HJuLgqRytjBZgwkCEOsA1EKCYRsYErjcI0cnlOVHHXZl0yKRRTdIRppbEj8Lk38SYQiNBySqxoDa50upIUhIb1agiCcm3gXFLkDGsMY1qi5JS6iB2qsBGNTKG82xe/CRYCTaqW2kXKKUB3hDS2zlgDOfTuPhF/Ni6q9q6GpyPtRl3U779yqRVjzavXmB/YPai/eq7/H2fzzJdDsf9esPy/QIR3uXz73Jlc4ZhGIZhEgkSv8gJMaWNqiXirSe2i280F/YvfCoJ+2JLIXXBQrKn5vt9IUa+v7qtbo6WyyD2pyuj5XWgrkM7vj8UYikfTC6SiyS+YtAL3rvS/TfVh+jqY6nPj9VhzfDzguKi3QAU0lRXjN35sRBj3UGk7h8OVRvLtPRCmpWKDN6vVPRtupw8fblevIfw9O4LsQjkrLqSP7QqFfnqm3yw6HuqROLFHqw/sdMX1Zq27M1TfKkXu2MKGa4bxgNtOF5Bm2t3hByKRfRaq6rmlpTTvY5n0u4IURtUhFzujpphQvSWq87BnRHSBp6j6ubXu5ionob37bskZFxbE+a94FRlkq6+UdwZIZ5MGnS4qXJkT62UnRHizIXAzVmKTjfczXZASNRzUre52I5O/w2n/EKG60uq9jEKNX6n1DuyTNBZFq60fV1RQaummmHQlDXvUIU8Emey8vXlO6Hzy0iu+obXsXwMsBy01b5UDWNdamNeXchV29FnzfQPgRr0evpB6zoqBeE3Fy/0fuufhdeKvaX8sRN9V2A/Klx83XHbpZ97KwstvxGsD4meP2r46396LWgY1oQ3bYxKO7MYhmEYhmEYhmEYhmEYhmEYhmEYhmEYhtkF/gMowJs3YI2tmAAAAABJRU5ErkJggg=="
            },
            "extension_attributes": {
                "video_content": {
                    "media_type": "external-video",
	            "video_provider":"youtube",
                    "video_url": "https://www.youtube.com/watch?v=FVFPRstvlvk",
                    "video_title": "Video title",
                    "video_description": "Video description",
                    "video_metadata": "Video meta"
                }
            }
        }
    ]
  },
  "saveOptions": true
}

**Comment by itwebdevgh** [comment]
@magento give me 2.3.1 instance

**Comment by magento-engcom-team** [comment]
Hi @itmsenior. Thank you for your request. I'm working on Magento 2.3.1 instance for you

**Comment by magento-engcom-team** [comment]
Hi @itmsenior, here is your [Magento instance](https://i-23194-2-3-1.instances.magento-community.engineering).
Admin access: [https://i-23194-2-3-1.instances.magento-community.engineering/admin](https://i-23194-2-3-1.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by itwebdevgh** [comment]
Hi @krishprakash I've just confirmed this issue on both 2.3.1 and develop instance.
You need to add the video to a product via the api, then remove that video in the admin and add it again via the API and then you will see the outcome I've mentioned.

Can you please retry with the above steps and re-open the issue for us? I'll update the steps to reproduce in the original post.

**Edit**: It appears you don't even need to delete the video. Just re-send the same json again and the video data will be lost just leaving the thumbnail.

**Comment by krishprakash** [comment]
Hi @itmsenior, 
You can not update the video using Post so please use the put method for update the video.
As send the same json request again it should be throw the error but it doesn't and remove the video.
So for that please update the description.

**Comment by itwebdevgh** [comment]
Hi @krishprakash, whether I use POST or PUT, the video information is not saved if the product has or has had a Youtube video.
I've updated my ticket title and description.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @krishprakash 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-17437`, `MC-17438` were created

**Issue Available**: @krishprakash, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @itmsenior. 

Thank you for your report and collaboration!

The issue was fixed by Magento team. 

The fix will be available with the upcoming `2.3.5` release.

**Comment by magento-engcom-team** [comment]
Hi @itmsenior. 

Thank you for your report and collaboration!

The issue was fixed by Magento team. The fix was delivered into `magento/magento2:2.4-develop` branch(es).
Related commit(s):
 - [6724a91cf01a6b31a56ac0967206d6a365c840cd](https://github.com/magento/magento2/commit/6724a91cf01a6b31a56ac0967206d6a365c840cd)

The fix will be available with the upcoming `2.4.0` release.
